r/tests/testthat/test-dataset.R

```diff
@@ -304,6 +305,25 @@ test_that("Dataset and query print methods", {
   )
 })
 
+expect_scan_result <- function(ds, schm) {
```

Comment:
Give a comment here explaining why this function exists (or, if you remove Dataset$create, you don't need this factoring-out anymore)